modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/utils/getAPISourceItems.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {fetch} from 'frontend-js-web';
+import fuzzy from 'fuzzy';
+
+export default async function getAPISourceItems({
```

Comment:
There are two problems with this util:
1. It's not in the right scope, it should not be in top utils dir, but in smallest applicable scope.
2. More importantly, it should not be used from two files when we only use it in one place in UI. Something has gone wrong with separation of concerns.